Makefile

```diff
@@ -56,6 +59,17 @@ mnesia_backup:
 		-e backup_suffix=$(BACKUP_SUFFIX) \
 		mnesia_backup.yml
 
+~/.ssh/id_ae_infra_ed25519:
```

Comment:
I tested this creates priv and pub key as separate files (id_ae_infra_ed25519 and id_ae_infra_ed25519.pub). Ok.